### ISSUE TITLE
[RDM] Alternate Raise Feature

### ISSUE THREAD

**Issue by Iaotle** [Nik-Potokar/XIVSlothCombo#476]
Honestly the alternate feature is always superior to the normal one imo, but we need to add it to RDM

**Comment by Tsusai** [comment]
I agree with the alternate versions on healers being the better code...but when I recently wrote the option for RDM, I played with writing both, and I decided with implementing the Swiftcast (non alt) based one from healers, because of two conditions. 

1st issue: Healers unlock their raise before swiftcast (Sage is raise 12 vs swift 18). The alternate Raise replace with Swift (I think) works great here with healers. Didn't feel that way with Red Mage, as theirs unlocks at 63. So Swiftcast already exists, and overriding it for verraise when it's on cooldown or or not while sync'd was real neat code.
2nd issue: Dualcast proc. Was easier to write in

But I did take bits from Alternate make the implemented feature a bit better

Have you tried it? I ran a Nier Tower and it was gloriously nice.
